dcrtimed/backend/filesystem/fsck.go

```diff
@@ -18,7 +18,7 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/decred/dcrd/txscript"
-	"github.com/decred/dcrdata/v4/api/types"
```

Comment:
api/types has since been made into it's own module, keeping the module sums under control